bin/env.sh

```diff
@@ -111,6 +111,9 @@ if [ ! -d "${TOPQ_CONDA_PATH}" ] ; then
 	conda clean -t -y
   # give the group write access
   chmod g+r -R ${TOPQ_CONDA_PATH}
+  # Clear the PYTHONPATH
+  source deactivate
+  source activate ntp
```

Comment:
Is NTupleProduction/python still in the python path after this?

The reset should got at the very top of the script: `unset PYTHONPATH`